streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionVertexState.java

```diff
@@ -0,0 +1,25 @@
+package org.ray.streaming.runtime.core.graph.executiongraph;
+
+import java.io.Serializable;
+
+/**
+ * Vertex state.
+ */
+public enum ExecutionVertexState implements Serializable {
+
+  /**
+   * execution vertex state enum
+   */
+  TO_ADD(1, "TO_ADD"),
```

Comment:
What's the meaning of `TO_ADD`? why we need this? same for `TO_DEL `

Reply:
This is corresponding to 'addWorker' and 'destroyWorker' later in job scheduler.

Reply:
Done.